src/igvfd/tests/test_upgrade_auxiliary_set.py

```diff
@@ -0,0 +1,10 @@
+import pytest
+
+
+def test_auxiliary_set_upgrade_1_2(upgrader, auxiliary_set_v1):
+    value = upgrader.upgrade(
+        'auxiliary_set', auxiliary_set_v1,
+        current_version='1', target_version='2')
+    assert value['schema_version'] == '2'
```

Comment:
please compare the content of publication_identifiers and refrences in value, it is not sufficient to check for presence if the content was lost